services/slam/builtin/builtin_test.go

```diff
@@ -1059,6 +1059,10 @@ func TestEndpointFailures(t *testing.T) {
 	test.That(t, pc, test.ShouldBeNil)
 	test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error getting SLAM map")
 
+	fullBytes, err := slam.GetPointCloudMapFull(context.Background(), svc, "hi")
```

Comment:
We have several instances of testing where GetPointCloudMapFull is used instead of GetPointCloudMapStream. All errors associated with GetPointCloudMapStream are passed through, so testing is unaffected and more compact. However, if we want to call GetPointCloudMapStream directly testing would look like this:

```
	f, err := svc.GetPointCloudMapStream(context.Background(), "hi")
	test.That(t, err, test.ShouldBeNil)
	test.That(t, f, test.ShouldNotBeNil)
	chunk, err := f()
	test.That(t, err.Error(), test.ShouldContainSubstring, "Unimplemented desc")
	test.That(t, chunk, test.ShouldBeNil)
```